Uni_life_tracker/app/src/main/java/com/generic/plannr/Gateways/UserGateway.java

```diff
@@ -0,0 +1,262 @@
+package com.generic.plannr.Gateways;
+
```

Comment:
Isn't this the same as the userdatabasehelper?

Reply:
Not exactly since it's interacting with the new DatabaseClient class which is the only helper. Its only job is to create the database, and the methods that interact with the database are now placed in these gateways. This was done to better model Clean Architecture and allow decoupling / remove hard dependencies.